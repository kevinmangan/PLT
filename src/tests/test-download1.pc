
google = download("http://www.google.com");
lines = List.split(google, '\n');
print(lines[0]);
