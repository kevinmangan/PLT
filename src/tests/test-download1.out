<!doctype html><html itemscope="" itemtype="http://schema.org/WebPage"><head><meta content="Search the world's information, including webpages, images, videos and more. Google has many special features to help you find exactly what you're looking for." name="description"><meta content="noodp" name="robots"><meta itemprop="image" content="/images/google_favicon_128.png"><title>Google</title><script>(function(){